plugins/modules/ovirt_event.py

```diff
@@ -76,7 +76,7 @@
         description:
             - "The id of the template associated with this event."
         type: str
-    user:
+    ansible.builtin.user:
```

Comment:
How is this related to ansible? Isn't `user` just an id of a user associated within this event and as such it has no relation to https://docs.ansible.com/ansible/latest/collections/ansible/builtin/user_module.html ,right?

Reply:
removing this resolves https://github.com/oVirt/ovirt-ansible-collection/issues/546

Reply:
Sorry, too broad automatic change. Same for others below.

Reply:
> removing this resolves #546

Ok, I'll try to rebase on a revert for it.